awx/ui_next/src/components/AdHocCommands/AdHocDetailsStep.jsx

```diff
@@ -70,6 +83,9 @@ function CredentialStep({ i18n, verbosityOptions, moduleOptions }) {
               id="module_name"
               data={moduleOptions || []}
```

Comment:
We could include a form select placeholder by adding it inline:
```suggestion
              data={[
                {
                  value: '',
                  key: '',
                  label: i18n._(t`Select a module`),
                  isDisabled: true,
                },
                ...moduleOptions
              ]}
```